applications/Chat/evaluate/README.md

```diff
@@ -1,182 +1,260 @@
+|        Chat         | Conduct multiple rounds of dialogue, the capability of understanding and memorization of previous rounds of dialogue                     |
+|       Open QA       | Given an open question, the capability of answering questions in opened-ended way                                                        |
+|      Closed QA      | Given a closed question, the capability of answering questions with limited scope (such as single/multiple choice question)              |
+|    Brainstorming    | Given a question requiring divergent answers, the capability of divergent answering and listing in points                                |
+|     Generation      | Given generation task, the capability of generating in high quality and human-written way (such as writing an email)                     |
+|      Rewriting      | Given rewriting task, the capability of rewriting sentences to meet task requirements (such as active and passive switches, translation) |
+|   Classification    | Given classification task, the capability of accurate classification                                                                     |
+|     Extraction      | Given extraction task, the capability of extracting required information                                                                 |
+|    Summarization    | Given a paragraph or passage, the capability of summarization                                                                            |
+
+To better understand each evaluation category, here are some prompt examples provided.
+
+
+| Evaluation Category | Chinese Example                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | English Example                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
Do we need to show English examples? The evaluation pipeline for English capability is under preparation.

Reply:
@TongLi3701 suggest English examples can be put. We are preparing for the English capability evaluation and given some prompt examples to explain different categories, I think it can be acceptable?